### PR TITLE
Multiple small UI enhancements

### PR DESCRIPTION
**Pull request by rabuckley**
The changes are fairly minor, but the diffs are quite big because of the changed markup (e.g. using lists). If you'd like the changes in this PR, I would imagine it would be easiest to pull locally and click around.

### CODE REVIEW

**Comment by KristofferStrube** [comment]
A lot of great changes. I will need a bit to go through them all. I will get back to you tomorrow. 😊

File: src/KristofferStrube.EditorConfigWizard/Shared/HighLightedCode.razor [hunk 1]
```diff
@@ -26,8 +30,28 @@
...
+            var sb = new StringBuilder();
+            
+            if (BorderHoverColor is not null)
+            {
+                hasHover = true;
+                sb.Append($"{BorderHoverColor.Value.ToString().ToLower()}-hover ");
+            }
+            if (BorderColor is not null)
+            {
+                sb.Append($"{BorderColor.Value.ToString().ToLower()}-hover ");
```
Comment (KristofferStrube) [inline src/KristofferStrube.EditorConfigWizard/Shared/HighLightedCode.razor]: This should simply be the color without the "-hover" part.

Comment (rabuckley) [reply src/KristofferStrube.EditorConfigWizard/Shared/HighLightedCode.razor]: Good catch, thanks. Fixed and rebased.

**Comment by KristofferStrube** [comment]
Now I've reviewed it. I think it was a lot of small good changes. I especially like the places where you use semantic tags and that you made the transition faster. It really is a lot snappier now.

**Pull request merged**